.github/workflows/tests.yml

```
@@ -454,6 +454,9 @@ jobs:
             -   name: Pre-merge naming/package consistency gate
                 run: uv run --frozen --no-build python -m scripts.engineering.qa check-naming-pkg --check
 
+            -   name: Pre-merge naming convention audit
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)